packages/core/src/RPA/core/webdriver.py

```diff
@@ -21,94 +27,60 @@
     "Darwin": ["Chrome", "Safari", "Firefox", "Opera"],
     "default": ["Chrome", "Firefox"],
 }
+AVAILABLE_DRIVERS = {
+    # Driver names taken from `webdrivermanager` and adapted to `webdriver_manager`.
+    "chrome": ChromeDriverManager,
+    "firefox": GeckoDriverManager,
+    "gecko": GeckoDriverManager,
+    "mozilla": GeckoDriverManager,
+    "opera": OperaDriverManager,
+    # NOTE: There's no specific `EdgeDriverManager` with this manager.
+    "edge": EdgeChromiumDriverManager,
```

Comment:
Not sure what is the best approach here. Edge referred to the old edge which isn't chromium-based. We're dropping the support here, but I'm a bit on the fence if we should just remove the alias or point towards chromium edge. New users would probably expect to get chromium edge instead of the old one, maybe.

Reply:
Thank you for the observation!

Also an interesting question is if the `selenium.webdriver.Edge` at least works with the chromium-edge driver. (haven't tested)

And there was a bug here proving that nobody used the Chromium flavour of Edge because we had `edgechromium` expecting for a browser name "EdgeChromium" which will fail the `getattr` on the `webdriver` module because we actually have the "ChromiumEdge" available (reversed lexical atoms) as seen [here](https://github.com/SeleniumHQ/selenium/blob/selenium-3.150.0/py/selenium/webdriver/__init__.py#L26). (in both Selenium 3 and 4)
So all our Edge users we'll be redirected to ChromiumEdge until this upstream [Issue](https://github.com/SergeyPirogov/webdriver_manager/issues/431) gets sorted out.

Reply:
If I remember correctly, non-chromium Edge requires a completely different webdriver, and it had to be installed via the Windows add/remove features menu.

Reply:
Possibly why it's not supported by webdriver-manager.

Reply:
Since in Selenium 4 `Edge` became the same as `ChromiumEdge` I feel is safe to keep the same driver manager for both and assume than `edge` is just an alias for `chromiumedge` which is the only one currently supported now.